zone/zonedb.cpp

```diff
@@ -3770,15 +3770,25 @@ void ZoneDatabase::SavePetInfo(Client *client)
 		petinfo = client->GetPetInfo(pet);
 		if (!petinfo)
 			continue;
+		
+		std::string Orig = GetOriginalPetName(pet,client->CharacterID());
+		if (Orig == "") 
```

Comment:
Can check against std::string::empty instead. (Technically this is creating another string object currently, no idea if the compiler is smart enough to avoid it though)

Reply:
4 or so methods could be used, but yeah looking at emu code, .empty is used much more than anything else. It uses a size comparative, so possibly slightly less overhead.
Switched to s.empty() for consistency.